For #154 Padding issue fixed and reduced repeated code

mohitsoni-dev opened the pull request.

mohitsoni-dev committed:
Padding issue fixed and reduced repeated code

arteevraina commented:
@mohitsoni-dev please update your PR with some description. It can include stuff like which issue you fixed and what changes you made. Screenshots if necessary. :)

mohitsoni-dev commented:
Reduced repeated code by extracting the Expanded widget and added needed padding to the text. Extracted widget:-
```
class SettingText extends StatelessWidget {
  SettingText({@required this.text});
  final String text;
  @override
  Widget build(BuildContext context) {
    return Expanded(
      flex: 3,
      child: Padding(
        padding: const EdgeInsets.all(8.0),
        child: Text(
          text,
          style: TextStyle(
            height: 1,
            fontSize: 17,
          ),
        ),
      ),
    );
  }
}
```
Before & After
<img src="https://user-images.githubusercontent.com/59525097/110577053-dcce2e80-8187-11eb-8c48-be9e83dd2e48.jpg" alt="before" width=250> <img src="https://user-images.githubusercontent.com/59525097/110577294-63830b80-8188-11eb-9a7b-d8da35e253e5.jpeg" alt="before" width=250>

Please inform me if any changes are needed in this PR. Thanks.

arteevraina commented on lib/settings.dart:
```diff
@@ -112,28 +106,25 @@ class _SettingsScreenState extends State<SettingsScreen> {
-                          ),
+                          SettingText(
+                              text:
+                                  'When auto-capturing , let me adjust borders after each scan'),
```
reformat this as well.

arteevraina commented on lib/settings.dart:
```diff
@@ -148,29 +139,24 @@ class _SettingsScreenState extends State<SettingsScreen> {
-                            ),
-                          ),
+                          SettingText(
+                              text: 'Run text recognition on saved pdf'),
```
reformat this.

arteevraina commented on lib/settings.dart:
```diff
@@ -360,4 +308,25 @@ class _SettingsScreenState extends State<SettingsScreen> {
\ No newline at end of file
+}
+
+class SettingText extends StatelessWidget {
```
I would suggest adding this class in a separate file. But would like to have @anushbhatia 's view on this as well.

arteevraina reviewed (commented):
Just a few changes. Other than that PR looks good. Nice Work :confetti_ball:

mohitsoni-dev commented:
Understood. I'll make the suggested changes and make update the PR. Thanks.

mohitsoni-dev committed:
Moved SettingText class to a separate dart file

mohitsoni-dev commented:
It seems there is some problem with the lint test maybe it's outdated. @arteevraina please respond to this. Thanks.

arteevraina commented:
`flutter analyze`  is failing because there are a lot of warnings in the code itself that's why it's failing. Not specifically about your code. I think this should be dealt in a separate issue. Ref: #161
